doc/make.jl

```diff
@@ -166,49 +166,14 @@ makedocs(
     assets = ["assets/julia-manual.css", ]
 )
 
-# This overloads the function in Documenter that generates versions.js, to include
-# v1/ in the version selector, instead of stable/.
-#
-# The function is identical to the version found in Documenter v0.19.6, except that
-# it includes "v1" instead of "stable".
-#
-# Original:
-# https://github.com/JuliaDocs/Documenter.jl/blob/v0.19.6/src/Writers/HTMLWriter.jl#L481-L506
-#
-import Documenter.Writers.HTMLWriter: generate_version_file
-function generate_version_file(dir::AbstractString)
-    named_folders = ["v1", "latest"]
-    tag_folders = []
-    for each in readdir(dir)
-        each == "v1" && continue # skip the v1 symlink
-        occursin(Base.VERSION_REGEX, each) && push!(tag_folders, each)
-    end
-    # sort tags by version number
-    sort!(tag_folders, lt = (x, y) -> VersionNumber(x) < VersionNumber(y), rev = true)
-    open(joinpath(dir, "versions.js"), "w") do buf
-        println(buf, "var DOC_VERSIONS = [")
-        for group in (named_folders, tag_folders)
-            for folder in group
-                println(buf, "  \"", folder, "\",")
-            end
-        end
-        println(buf, "];")
-    end
-end
-
 # Only deploy docs from 64bit Linux to avoid committing multiple versions of the same
 # docs from different workers.
 if "deploy" in ARGS && Sys.ARCH === :x86_64 && Sys.KERNEL === :Linux
-    # Since the `.travis.yml` config specifies `language: cpp` and not `language: julia` we
-    # need to manually set the version of Julia that we are deploying the docs from.
-    ENV["TRAVIS_JULIA_VERSION"] = "nightly"
-
     deploydocs(
-        julia = "nightly",
         repo = "github.com/JuliaLang/julia.git",
         target = "_build/html/en",
         dirname = "en",
-        deps = nothing,
-        make = nothing,
+        devurl = "1.1-DEV",
```

Comment:
Having this be lowercase would look better in the URL in my opinion (`/en/1.1-dev/`).

Reply:
We have
```
julia> VERSION
v"1.1.0-DEV.361"
```
though?

Reply:
They don't necessarily have to agree I'd say. In any case, I think it would be fine to have uppercase in the selector, but we can't separate it from what the URL is at the moment.

Reply:
> but we can't separate it from what the URL is at the moment.

Why not? We can have `devurl = "v1.1-dev"` and `"v1.1-DEV" => "v1.1-dev"` in `versions`.